### PR TITLE
fix(PPDSC-2480): load correct font urls

### PR DESCRIPTION
**Pull request by mstuartf**
[PPDSC-2480](https://nidigitalsolutions.jira.com/browse/PPDSC-2480)

**What**

1. Background - why this is needed
Font file not loading for Virgin so incorrect font displaying in Storybook theme checker
2. What did you do
- Old Montserrat font link was invalid
- Instead of hardcoding the new one, add a link element to load a stylesheet that imports the fonts
- This matches what Virgin do on their site
3. What does the reviewers should expect
Virgin fonts display correctly in theme checker

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2480-virgin-fonts
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2480-virgin-fonts/storybook

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -325,16 +325,13 @@
...
   /* This is to override the size of Canvas in docs pages */
   .docs-story .innerZoomElementWrapper > div > div > div {
     position: relative;
   }
 </style>
+
+<link
+  rel='stylesheet'
+  href='https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,600;0,700;1,400;1,700&amp;display=swap'
+/>
```
Comment (mutebg) [inline .storybook/preview-head.html:337]: to me personally, that's the simplest solution!

**Review by mutebg — APPROVED** [review]

File: .storybook/preview-head.html (hunk 1 above)
Comment (Xin00163) [reply .storybook/preview-head.html:337]: Pretty neat. How did you find this? @mstuartf

**Review by Xin00163 — APPROVED** [review]

File: .storybook/preview-head.html (hunk 1 above)
Comment (mstuartf) [reply .storybook/preview-head.html:337]: I looked at the Virgin site and that's how they load their fonts there. I think the reason that this stopped working is that the old google fonts url was out of date / invalid. But this way we shouldn't have this problem again.

**Pull request merged**